packages/core/database/src/metadata/metadata.ts

```diff
@@ -14,6 +14,13 @@ export interface Meta extends Model {
 }
 
 export class Metadata extends Map<string, Meta> {
+  // TODO: we expose the global identifiers in this way so that in the future we can instantiate our own
+  // However, it should NOT be done until all the methods used by metadata can be part of this metadata object
+  // and access this one; currently they all access the global identifiers directly.
+  get identifiers() {
```

Comment:
@alexandrebodin I've reduced the entire public footprint down to this. And like I've written in the comment, in a later refactor this should be completely encapsulated with its own identifiers object, but that would require refactoring every method called by metadata to be part of a metadata instance so they could access the local one, otherwise we're back at the problem of needing to pass an additional argument to every method here.

But at least like this, the public interface is as minimal and correct as it can be and it won't be a breaking change if we want to refactor later.